SPECS/libtpms/libtpms.spec

```diff
@@ -0,0 +1,345 @@
+Name:           libtpms
```

Comment:
I think we have missed to add an entry to licenses.

Typically something like this helps:
```bash
./toolkit/scripts/license_map.py --no_check --update --remove_missing \
        SPECS/LICENSES-AND-NOTICES/data/licenses.json \
        SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md \
        SPECS SPECS-EXTENDED SPECS-SIGNED
```

Reply:
the command you pasted didn't change any files in my repo.  what should it be doing?

Reply:
maybe this isn't recognized by our tooling?

`License:        BSD and TCGL`

Reply:
A representative PR:
- https://github.com/microsoft/CBL-Mariner/pull/6063/files#diff-74e8b3639b8ff8af22317c85e1868d1bd81f0f6bd24011a64af4058417c370eeR670

A new package has two additional entries:
- licenses.json
- LICENSE-MAP.md

Reply:
thanks!  is it as simple as: if the spec is imported from fedora, the package goes into the fedora section?

Reply:
Yes!